src/resources/js/commerce/gateway/toggler.js

```diff
@@ -113,7 +114,10 @@ tribe.tickets.commerce.gateway = tribe.tickets.commerce.gateway || {};
 	 * @param {Element} gateway Gateway element to show.
 	 */
 	obj.toggler.showGateway = gateway => {
```

Comment:
I'm not entirely sure why we use the `obj.toggler` prefix. We could remove the `toggler` part here, and addit in the obj (the last line, make it `})( jQuery, tribe.tickets.commerce.gateway.toggler );`)

We'll also need to define the toggler beforehand, up in the code: `tribe.tickets.commerce.gateway.toggler = tribe.tickets.commerce.gateway.toggler || {};`

I think we should keep doing that for consistency between libraries.

Reply:
Good idea. Thanks!